boards/arduino-uno/examples/uno-blink.rs

```diff
@@ -0,0 +1,35 @@
+#![no_std]
+#![no_main]
+
+extern crate panic_halt;
+use arduino_uno::prelude::*;
+
+#[no_mangle]
+pub extern fn main() -> ! {
+    let dp = arduino_uno::Peripherals::take().unwrap();
+
+    let mut delay = arduino_uno::Delay::new();
+    let mut pins = arduino_uno::Pins::new(
+        dp.PORTB,
+        dp.PORTC,
+        dp.PORTD,
+    );
+
+    // Digital pin 13 is also connected to an onboard LED marked "L"
+    let mut led0 = pins.d13.into_output(&mut pins.ddr);
+
+    led0.set_high().void_unwrap();
+
+    let mut led = led0.downgrade();
```

Comment:
If you have just one led, there is no need to downgrade it.  `downgrade()` is only necessary if you want to store multiple pins in an array (as each pin by default has its own type).

It also introduces a runtime overhead because it needs to discern which port the pin is connected to.

Reply:
Cool, done